server/main.lua

```diff
@@ -71,14 +88,17 @@ RegisterNetEvent('qb-banking:createNewCard', function()
     local src = source
     local xPlayer = QBCore.Functions.GetPlayer(src)
 
```

Comment:
```suggestion
if not xPlayer then return end
```
A nil check for rare-cases where a cheater or a person is not registered in qb-core either through a bug or a weird cheat

Reply:
If you could, please add this check to every event there is